app/src/main/java/com/example/daggersampleapplication/MainActivity.kt

```diff
@@ -1,8 +1,9 @@
 package com.example.daggersampleapplication
 
+import android.content.Intent
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mattukouta/DaggerSample/blob/3469102ef3fd38db1a6087a8e8bbbe4c3cbd7710/app/src/main/java/com/example/daggersampleapplication/MainActivity.kt#L3"></span>Imports must be ordered in lexicographic order without any empty lines in-between</td>
    </tr>
  </tbody>
</table>